src/util.h

```diff
@@ -634,7 +634,7 @@ inline QString svrRegStatusToString ( ESvrRegStatus eSvrRegStatus )
         return QCoreApplication::translate ( "CServerDlg", "Requirements not fulfilled" );
     }
 
-    return QString ( QCoreApplication::translate ( "CServerDlg", "Unknown value " ) ).append ( eSvrRegStatus );
```

Comment:
Translations affected.

Reply:
Yep. If this is merged for 3.9.0 it should be no problem though, right?
If you think it's ok to merge for 3.8.2, then we could manually fix the .ts files in this case or ignore it for now. The latter should be OK as well as this is a code path which is never supposed to be taken ("last resort fallback").

Reply:
I'd stick with 3.9.0 for this.